nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/summary/ui/process-group-status-listing/process-group-status-table/process-group-status-table.component.ts

```diff
@@ -98,23 +98,23 @@ export class ProcessGroupStatusTable extends ComponentStatusTable<ProcessGroupSt
 
     private versionedFlowStateMap: { [key: string]: { classes: string; label: string } } = {
         STALE: {
-            classes: 'fa fa-arrow-circle-up stale',
+            classes: 'fa fa-arrow-circle-up nifi-warn-light',
```

Comment:
stale looks good on this background with the nifi-warn-light color. i think it is just the case on the canvas when it is on the light blue background that this color looks a bit washed out.

Reply:
You mean in the PG header? Yea I agree. We will address this in https://issues.apache.org/jira/browse/NIFI-13011.